src/app/main/login/login.component.ts

```diff
@@ -26,13 +28,21 @@ export class LoginComponent implements OnInit {
     ngOnInit() {
     }
 
-    refresh(status: boolean) {
+    refresh() {
+        // check if a session is active
+        if (this._session.getSession()) {
+            this._router.navigate(['dashboard']);
+        }
```

Comment:
After successful login the user will always be redirected to the dashboard page, right? 
Why not keeping `this._router.navigate([this.returnUrl]);`? In case a user bookmarked his project page e.g. `admin.dasch.swiss/project/1111/info` but (s)he's no longer logged-in, then it would be nice to be redirected to this page after logging-in. 
But it's just a detail... maybe not important 😉

Reply:
this is why I redirect explicitly to the dashboard, after logging in, I get this page (which I do not want):
<img width="1551" alt="Screenshot 2021-01-27 at 15 56 58" src="https://user-images.githubusercontent.com/29532462/106009114-4b639b00-60b8-11eb-8152-c563e73c82da.png">

Reply:
haha...yes true. The root path `/` was handled in the main component before. This component showed the landing page or the dashboard. Yes, as I said "it's just a little detail". I can live with it.
Another solution would have been to integrate the login page instead of the landing page. I.e. that still the main component handles the redirection. 
But it's fine 😉